interface/super/manage_site_files.php

```diff
@@ -99,6 +100,42 @@
   }
 
 }
+
+/**
+ * Thumbnails generator
+ * generating  thumbnail image to all images files from documents table
+ */
+
+switch((int)$GLOBALS['document_storage_method']){
+    case 0:
+        $storage_method = 'on Disc';
+        break;
+    case 1:
+        $storage_method = 'in CouchDB';
+        break;
+}
+
+if(isset($_POST['generate_thumbnails'])) {
+
+    $thumb_generator = new ThumbnailGenerator();
+    $results = $thumb_generator->generate_all();
+
+    $thumbnail_msg = "<p style='color: green'>" . xl('Generated thumbnail(s)') . " : {$results['sum_success']}</p>";
```

Comment:
for new code, need to escape correctly:
``````
$thumbnail_msg = "<p style='color: green'>" . xtl('Generated thumbnail(s)') . " : " . text($results['sum_success']) . "</p>";